pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -53,9 +54,7 @@ public HelixBrokerStarter(PinotConfiguration brokerConf, String clusterName, Str
   private static PinotConfiguration applyBrokerConfigs(PinotConfiguration brokerConf, String clusterName, String zkServers, @Nullable String brokerHost) {
     brokerConf.setProperty(Helix.CONFIG_OF_CLUSTER_NAME, clusterName);
     brokerConf.setProperty(Helix.CONFIG_OF_ZOOKEEPR_SERVER, zkServers);
-    if (brokerHost == null) {
-      brokerConf.clearProperty(Broker.CONFIG_OF_BROKER_HOSTNAME);
-    } else {
+    if (!Strings.isNullOrEmpty(brokerHost)) {
```

Comment:
shall we also keep the logic to delete broker host? Or we expect this host/port to always be set?

Reply:
Based on code reading, I don't see a use case that we need to clear the values. cc @Jackie-Jiang for double checking